q2_types/feature_data/_format.py

```diff
@@ -100,34 +100,51 @@ class TSVTaxonomyFormat(model.TextFileFormat):
     """
     HEADER = ['Feature ID', 'Taxon']
 
-    def sniff(self):
+    def _check_tsv_tax_format(self, root, n=None):
         with self.open() as fh:
             data_lines = 0
             header = None
-            while data_lines < 10:
-                line = fh.readline()
+
+            file_ = enumerate(fh) if n is None else zip(range(n), fh)
+
+            for iter, line in file_:
```

Comment:
`iter` is a [builtin](https://docs.python.org/3/library/functions.html#built-in-functions) in python - let's rename this variable to something else (could be `i`, for example).

Reply:
I replaced `iter` with `i` so that a builtin is not used.